code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -0,0 +1,404 @@
+/mob/living/simple_animal/bot/honkbot
+	name = "\improper Honkbot"
+	desc = "A little robot. It looks happy with its bike horn."
+	icon = 'icons/mob/aibots.dmi'
+	icon_state = "honkbot0"
+	density = FALSE
+	anchored = FALSE
+	health = 20
+	maxHealth = 20
+	damage_coeff = list(BRUTE = 0.5, BURN = 0.7, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	pass_flags = PASSMOB
+
+	radio_key = /obj/item/device/encryptionkey/headset_service //doesn't have security key
+	radio_channel = "Service" //Service
+	bot_type = SEC_BOT
+	model = "Honkbot"
+	bot_core_type = /obj/machinery/bot_core/honkbot
+	window_id = "autosec"
+	window_name = "Honkomatic Bike Horn Unit v1.0.1"
+	allow_pai = 1 //Damn right we'll pAI these
+	data_hud_type = DATA_HUD_SECURITY_ADVANCED // show jobs
+
+	var/honksound = 'sound/items/bikehorn.ogg' //customizable sound
+	var/goldenbikehorn = 0 //placeholder
+	var/spam_flag = 0
+	var/cooldowntime = 40
+	var/cooldowntimehorn = 10
+	var/cooldowntimeEmag = 5
+	var/mob/living/carbon/target
+	var/oldtarget_name
+	var/target_lastloc //Loc of target when arrested.
+	var/last_found //There's a delay
+	var/threatlevel
+	var/declare_arrests = 0 // speak, you shall not, unless to Honk
+	var/idcheck = 1 //Chases unknowns
+	var/fcheck = 1 //And armed people
+	var/check_records = 1 //Doesn't care about criminals
+	var/arrest_type = 0
+	var/weaponscheck = 1
+
+/mob/living/simple_animal/bot/honkbot/Initialize()
```

Comment:
already mentioned before on irc but replace New() with Initialize() and ..() with . = ..()

Reply:
Done.